src/pam_unix_ng-passwd.c

```diff
@@ -355,6 +365,10 @@ unix_chauthtok(pam_handle_t *pamh, int flags, struct config_t *cfg)
 	    }
 	  else if (strlen(strempty(pass_new)) < (size_t)cfg->minlen)
 	    {
+	      /*
+	       * so root is exempt from the minimum password length?
+	       * isn't he let off the hook too easily? a warning at least?
+	       */
```

Comment:
That's why you should use additional PAM modules like pam_pwquality, which can also enforce this rules for root.
Strictly spoken that should not be part of pam_unix_ng, it's only here to make it easier to switch from pam_unix to pam_unix_ng, so we mimic the same behavior.